scss/components/_page.scss

```diff
@@ -0,0 +1,12 @@
+*,
+*:before,
+*:after {
+  box-sizing: border-box;
```

Comment:
this way all components styles will depends on the `page` component. And if I remove page component (or I would like to reuse another component in another app) -- styles will be broken

You can match only tags